frontend/src/app/App.tsx

```diff
@@ -335,6 +336,16 @@ const App = () => {
                   />
                 }
               />
+              <Route
+                path="upload-patients"
+                element={
+                  <ProtectedRoute
+                    requiredPermissions={canEditPeople}
```

Comment:
Should `requiredPermissions` also be set to `isSupportAdmin` for now? I might be misunderstanding how the permissions work here. It makes sense the element will be available only to support admins, just not sure how these two fields interact 👀